quarkchain/cluster/cluster_config.py

```diff
@@ -91,17 +91,21 @@ class SlaveConfig(BaseConfig):
     PORT = 38392
     WEBSOCKET_JSON_RPC_PORT = None
     ID = ""
-    CHAIN_MASK_LIST = None
+    FULL_SHARD_ID_LIST = []
 
     def to_dict(self):
         ret = super().to_dict()
-        ret["CHAIN_MASK_LIST"] = [m.value for m in self.CHAIN_MASK_LIST]
+        # format to hex
+        ret["FULL_SHARD_ID_LIST"] = [
+            "0x{:08x}".format(i) for i in self.FULL_SHARD_ID_LIST
+        ]
         return ret
 
     @classmethod
     def from_dict(cls, d):
         config = super().from_dict(d)
-        config.CHAIN_MASK_LIST = [ChainMask(v) for v in config.CHAIN_MASK_LIST]
```

Comment:
We should maintain backward compability of CHAIN_MASK_LIST.  A simple way may just translate CHAIN_MASK_LIST + number of shards to FULL_SHARD_ID_LIST

Reply:
done